collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -469,7 +470,7 @@ private void setupViewModels() {
         formEntryViewModel.getCurrentIndex().observe(this, index -> {
             if (index != null && Collect.getInstance().getFormController() == null) {
                 // https://github.com/getodk/collect/issues/5241
-                Timber.e("getCurrentIndex() firing with null getFormController()");
+                Timber.e("getCurrentIndex() firing with null getFormController(). The one stored in formEntryViewModel is %s", (formEntryViewModel.isFormControllerSet() ? "not null" : "null"));
```

Comment:
Maybe I'm wrong, but I don't think we want this to by dynamic or we'll end up with two events in Crashlytics?

Reply:
I added it just to make sure but I'm pretty sure in all cases it's null controller in the application class and not null in view model. Even if I'm wrong and we end up with two separate events that's also nothing very bad I guess.

Reply:
Seems ok to me. Either way I think we'll have to search/dig to find these non-fatals. Let's try it.